samples/dashboard/demos/dashboard-component/demo.mjs

```diff
@@ -1,165 +1,167 @@
 import Highcharts from 'https://code.highcharts.com/es-modules/masters/highmaps.src.js';
-
 import HighchartsComponent from 'https://code.highcharts.com/es-modules/Extensions/DashboardPlugins/HighchartsComponent.js';
 import DOMComponent from 'https://code.highcharts.com/es-modules/Dashboard/Component/HTMLComponent.js';
 import CSVStore from 'https://code.highcharts.com/es-modules/Data/Stores/CSVStore.js';
 
 Highcharts.AST.allowedAttributes.push('rows');
 
 const style = {
```

Comment:
Do we need it? It's simple so we could add it in a component's config.

Reply:
I added a task to asana that will be about cleaning each demo after everything is ready.